src/crccombine.h

```diff
@@ -0,0 +1,17 @@
+
+#include <stdint.h>
+
+
+#define USE_STATIC_COMBINE_CACHE 1
+// mask types
+typedef unsigned long long v2uq __attribute__ ((vector_size (16)));
+
+// final implementation
```

Comment:
No idea what this comment means, was this from the original file? It's not very clear now.

Reply:
will remove